light_curves/light_curve_generator.md

```diff
@@ -273,7 +268,7 @@ df_lc.append(df_lc_HCV)
 
 ```{code-cell} ipython3
 gaiastarttime = time.time()
-df_lc_gaia = gaia_get_lightcurve(sample_table, 1/3600., 0)
+df_lc_gaia = Gaia_get_lightcurve(sample_table, 1/3600., 0)
```

Comment:
This gives me the following error on the IRSA SP using the default kernel:

```
TypeError: TapConn.execute_tappost() takes from 3 to 4 positional arguments but 5 were given
```

I have the full traceback if that's helpful, but I'm guessing this is actually an astroquery bug... This comes from astroquery version 0.4.7.dev8948, which seems to be the version that is pre-installed in the image. @stargaser do you know why we're using a "dev" version here? This works fine if I revert to the latest non-dev version, astroquery==0.4.6.

Reply:
I wonder if we need to submit a bug report to astroquery. But I'll let @bsipocz weigh in first.

Reply:
Sorry I didn't see this sooner. I've adjusted my Github notifications to track this repository. I've responded in #173 that astroquery version 0.4.7.dev8948 is expected due to their continuous release model. @bsipocz can say more about that.

Reply:
yes, don't use 0.4.6. If the problem persist with the latest pypi available dev version, then do open a bug report, otherwise upgrade.

Reply:
OTOH, `Gaia_get_lightcurve` doesn't really adhere to snake case naming conventions, `gaia_get_lightcurve` looked better to me (but this is very much bike shedding from me)